app/src/test/kotlin/no/nav/aap/meldekort/KelvinIntegrasjonsPåFastsattDagTest.kt

```diff
@@ -160,11 +166,130 @@ class KelvinIntegrasjonsPåFastsattDagTest {
         }
     }
 
+    @Test
```

Comment:
Hadde vært fint med en test at ingen ting er uendret rundt 17. desember for de som har den andre fasen.